### PR TITLE
Infra & Prettier

### PR DESCRIPTION
**Pull request by vdegenne**

### CODE REVIEW

File: packages/api/src/DataBuilder.ts
Comment (VandeurenGlenn) [inline packages/api/src/DataBuilder.ts:1]: Why is the filename DataBuilder? prefer data-builder tbh

File: packages/api/src/api/github.ts [hunk 1]
```diff
@@ -1,72 +1,184 @@
...
-				{ headers: this.#headers }
-			);
-			
-			// @ts-ignore
-			if (response.status === 304) return cache.github.stats.cached[name]
-			if (response.status === 404) return {additions: 0, deletions: 0, total: 0};
-			// @ts-ignore
-			cache.github.stats.tags[name] = response.headers.get('ETag')
+		const headers = this.#headers;
+		headers.append('If-None-Match', cache.github.stats.tags[name]);
```
Comment (VandeurenGlenn) [inline packages/api/src/api/github.ts]: header(If-None-Match) is probably already set so we need to `set` & not `append`

File: packages/server/src/index.ts [hunk 2]
```diff
@@ -18,9 +18,8 @@ async function loadData() {
 }
 await loadData();
 
-
-// update cache every hour
-const job = await new CronJob('0 * * * *', updateCacheWithRemote);
+// fetch remotely every minute
```
Comment (VandeurenGlenn) [inline packages/server/src/index.ts:21]: 0 * * * *  is hourly

**Review by VandeurenGlenn — CHANGES_REQUESTED** [review]

File: packages/server/src/index.ts (hunk 2 above)
Comment (vdegenne) [reply packages/server/src/index.ts]: true!

File: packages/api/src/api/github.ts (hunk 1 above)
Comment (vdegenne) [reply packages/api/src/api/github.ts]: done

File: packages/server/src/index.ts (hunk 2 above)
Comment (VandeurenGlenn) [reply packages/server/src/index.ts]: [07ca979](https://github.com/VandeurenGlenn/blue_server/pull/20/commits/07ca979b9a840456813c84aa917b905dcd599848)

**Review by VandeurenGlenn — APPROVED** [review]

**Pull request merged**